src/freedv_api.c

```diff
@@ -1372,6 +1372,33 @@ int freedv_get_bits_per_modem_frame       (struct freedv *f) {return f->bits_per
 int freedv_get_rx_status                  (struct freedv *f) {return f->rx_status;}
 void freedv_get_fsk_S_and_N               (struct freedv *f, float *S, float *N) { *S = f->fsk_S[0]; *N = f->fsk_N[0]; }
 
+
+/*---------------------------------------------------------------------------*\
+
+  FUNCTIONS...: freedv_set_fmin_fmax
```

Comment:
:point_up: function name in doc

Reply:
ooops, done with https://github.com/drowe67/codec2/pull/309/commits/a45310aff1402d0bd86da31c4fd2d3812156b283